scripts/component

```diff
@@ -0,0 +1,109 @@
+        "url" : "https://github.com/micro-ROS/micro_ros_zephyr_module"
+    },
+    {
+        "name": "mbed_rtos",
+        "description": "micro-ROS module for Mbed RTOS: this package enables the integration of micro-ROS in any Mbed RTOS workspace.",
+        "url" : "https://github.com/micro-ROS/micro_ros_mbed"
+    },
+    {
+        "name": "nuttx_rtos",
+        "description": "micro-ROS module for NuttX RTOS: this package enables the integration of micro-ROS in any NuttX RTOS workspace.",
+        "url" : "https://github.com/micro-ROS/micro_ros_nuttx_app"
+    },
+    {
+        "name": "azure_rtos",
+        "description": "micro-ROS module for Microsoft Azure RTOS: this package enables the integration of micro-ROS in a Microsoft Azure RTOS workspace.",
+        "url" : "https://github.com/micro-ROS/micro_ros_azure_rtos_app"
+    },
+    {
+        "name": "tiva_c_series",
+        "description": "micro-ROS app for TI Tiva™ C Series: this package enables the integration of micro-ROS in a exas Instruments Tiva™ C Series.",
+        "url" : "https://github.com/micro-ROS/micro_ros_tivac_launchpad_app"
+    },
+    {
+        "name": "stm32cube",
+        "description": "micro-ROS utils for STM32CubeMX and STM32CubeIDE: this package enables the integration of micro-ROS in STM32CubeMX and STM32CubeIDE.",
+        "url" : "https://github.com/micro-ROS/micro_ros_stm32cubemx_utils"
+    },
+    {
+        "name": "platformio",
+        "description": "micro-ROS module for PlatformIO: this package enables the integration of micro-ROS in PlatformIO.",
+        "url" : "https://github.com/micro-ROS/micro_ros_platformio"
+    },
+    {
+        "name": "arduino",
+        "description": "Arduino IDE & CLI libraries for micro-ROS: this package enables the integration of micro-ROS in the Arduino IDE for some hardware platforms.",
+        "url" : "https://github.com/micro-ROS/micro_ros_arduino"
+    },
+    {
+        "name": "raspberry_pi_pico",
+        "description": "Raspberry Pi Pico SDK libraries for micro-ROS: this package enables the integration of micro-ROS in the Raspberry Pi Pico SDK.",
+        "url" : "https://github.com/micro-ROS/micro_ros_raspberrypi_pico_sdk"
+    }
+]
+
+def print_help(extended = False):
+    if extended:
+        print("Usage: python3 component.py <component_name>")
+        print("")
+        for component in COMPONENTS:
+            description = "\n\t".join(wrap(component["description"], MAX_CHARACTERS))
+            print("[{}] \n\t{}\n".format(component["name"], description))
+    else:
+        print("Usage: python3 component.py <component_name>")
+        print("Available components: {}".format(", ".join([c["name"] for c in COMPONENTS])))
+
+# No arguments
+if len(sys.argv) < 2:
+    print_help()
+    sys.exit(0)
+
+# User asks for help
+if "-h" in sys.argv or "--help" in sys.argv:
+    print_help(True)
```

Comment:
Should we add a specific help command? like 

` ros2 run micro_ros_setup component arduino --help`

which returns:
```bash
[arduino] 
	Arduino IDE & CLI libraries for micro-ROS: this package enables the integration
	of micro-ROS in the Arduino IDE for some hardware platforms.

        For usage instructions visit: https://github.com/micro-ROS/micro_ros_espidf_component#readme
```